providers/src/airflow/providers/google/provider.yaml

```diff
@@ -125,7 +125,7 @@ dependencies:
   - google-cloud-build>=3.22.0
   - google-cloud-compute>=1.10.0
   - google-cloud-container>=2.17.4
-  - google-cloud-datacatalog>=3.11.1
+  - google-cloud-datacatalog>=3.23.0
```

Comment:
Why do we have other providers changes here?

Reply:
That's why I had to redo this PR from the "apache" repository - because there are 6 backported commits, that I want to keep rather than squash (and having the PR from "apache" I can fast-forward the `providers-fab/v1-5` branch when the PR is approved rather than squash+merge - similarly as we do with `v2-10-stable` fastr-forwarding it to the tip of `v2-10-test` - this keeps the history of commits.

Reply:
This particular change with google datacatalog comes from this backported commit https://github.com/apache/airflow/pull/45185/commits/32e0e2ba7ae7002b645f9a36e7038adc85cacf2a

Reply:
See the failures it caused before -> https://github.com/apache/airflow/actions/runs/12458391437/job/34774059861#step:7:1025

Reply:
Got it.
I think it might be some what challanging doing such work if we had many breeze/ci changes in between.

Reply:
Not really - we do not have to backport all ci/dev changes - only those that are necesary to fix problems cause by some dependencies

Alternatively - if it will become too difficult we could make a change to only hard-code provider tests to only run "fab" and related providers tests. But for now I think it's less problematic to cherry-pick those PRs.